### DISCUSSION TITLE
2.2.1

### DISCUSSION THREAD

**Discussion by benhillis** [microsoft/WSL#11344] [General]
*  Enable DNS tunneling by default
*  Integrate DNS tunneling with Linux native Docker
*  Change the default reclamation mode to drop cache
*  Fix hang when the guest crashes during distro initialization
*  Retry when hitting errors that could indicate that an installation is in progress (solves #11282)
*  Remove the MSIX package before 'CreateShortcuts' runs during MSI installation (solves #11252,#10872)
*  Update Microsoft.WSL.Kernel to version 5.15.150.1-2
    * Enables CONFIG_HIDRAW and CONFIG_USB_HIDDEV
*  Update WIL to 1.0.240308.1

<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft/WSL/releases/tag/2.2.1'>2.2.1</a>.</em>

**Comment by crramirez** [comment]
May I ask what "WIL" is?

**Reply by kescherCode** [reply]
https://github.com/microsoft/wil

**Comment by Parsifa1** [comment]
In this version, it seems that the problem has appeared again..
https://github.com/microsoft/wslg/issues/1032
